src/index.ts

```diff
@@ -1,7 +1,7 @@
 import { Client } from 'pg'
 import split from 'split2'
 import { pipeline, TransformCallback, Transform } from 'stream'
-
+import os from 'os'
```

Comment:
I was playing around with turning lines into json and i used the `os` module to get hostname.  Decided it was better to just ignore them rather than fabricate JSON